WIP Inclusive Networking and Social Events subsection (#3380)
the-turing-way/the-turing-way: book/website/collaboration/inclusive-events/inclusive-events-socials.md

(cl-inclusive-events-social-and-networking-guide)=
# Inclusive Social and Networking Events

> Although your event's main content and sessions will be well thought out, interesting, and engaging for attendees, many attendees often speak highly of the social and collaborative environments that networking events bring to large events. 

Reviewer: I would delete this sentence as it doesn't contain that much info and start with the second sentence which provides a stronger argument.

Social events, dinners, and even standing in the lines for coffee can be some of the best times for attendees to get to know one another, identify potential collaborators, and spark ideas.

As an organizer, you should consider how to make these serendipitous opportunities for networking and socializing as accessible as possible for people with lived experience who are attending your event.

Depending on the size and format of your event, this may look like structured networking sessions, a social dinner, or a 15 minute social break at the start or middle of your event. 

Please consider some of the suggestions below, and ensure that you take the time to consider and address any barriers that your specific attendees may face in joining social settings.

(cl-inclusive-events-social-events-timing)=
## Choosing a Time for Networking 

> If your event schedule allows, ensure that you offer at least one networking or social event during the day. 

Reviewer: ```suggestion
Ideally at least one networking or social event will take place during the event. 
```

Author: I messed up reviewing the diff. This has been accepted and added to the file (albeit manually as I hit submit too quickly).

(cl-inclusive-events-social-events-accessibility)=
## Accessibility Requirements
Social events and networking should also have accessibility features like captioning or sign language interpretation, and include activities that do not restrict participation based on body type or ability.

For this reason, try to avoid "speed networking" where attendees have only 3-5 minutes to engage in conversation. 

For in-person events, this can be prohibitive for many people such as those with mobility impairments, people who require Sign Language interpreters or other Translators, and people with verbal processing differences. 
These same barriers affect people attending online events, as well as further negatively impacting those with low-bandwidth or internet connectivity issues. 

(cl-inclusive-events-name-tags)=
## Identification and Name Tags
If your event includes name tags or badges, ensure that you ask attendees for their preferred name, and display the name they provide. 

Consider including pronouns on attendees' badges to support attendees being gendered appropriately. 

> You may also want to consider using lanyards to display name tags or badges. Many clip-on name tags assume that people have a pocket on the front of their shirts or jackets. 

Reviewer: ```suggestion
You may also want to consider using lanyards to display name tags or badges. 
Many clip-on name tags assume that people have a pocket on the front of their shirts or jackets. 
```

However, many blouses, dresses, or shirts designed for women do not provide these pockets and therefore will not be able to easily display the name badge. 

> Clip-on badges can also be difficult for people with fine motor disabilities or an impairment that would restrict their ability to pin the clip onto their shirts. 

Reviewer: fuck clip on badges indeed.

(cl-inclusive-events-coloured-badges)=
### Traffic Light Badges
You may consider offering a "traffic light" coloured badge or name tag system that is available during social sessions or networking events. 
Traffic Light Systems can be a useful tool to set intent and expectations between attendees during social activities to make it clear which attendees are open to networking, and which would prefer to not engage in conversation. 

This can ensure that attendees who wish to not network are not surprised and forced to engage socially at your event, and that people who actively want to engage in conversation can easily identify others who also wish to actively network. 

> This traffic light system was used by LeadDev, a UK-based event organizer for their annual conference. 

Reviewer: Is there a link to LeadDev or the system?
